app/src/main/res/values/strings.xml

```diff
@@ -850,4 +850,12 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="user_profile_days_active">Days\nactive</string>
     <string name="user_profile_achievement_levels">Achievement\nlevels</string>
 
+    <string name="quest_surface_detailed_answer_impossible">Multiple surfaces at once…</string>
+    <string name="quest_surface_detailed_answer_impossible_confirmation">Are you sure that it is impossible to specify the surface? Note the \"Differs along the way\" answer option that allows to handle different surface sections. Please use \"Can\'t say\" if there is a single surface but it is not available as an answer.</string>
+    <string name="quest_surface_detailed_answer_impossible_description">Please briefly specify the nature of the surface here, for example "sandy with patches of cobblestone". The text length is limited to 255 characters.</string>
+    <string name="quest_surface_detailed_name_title">"What surface does the road %s have here? Please, select a specific one."</string>
+    <string name="quest_surface_detailed_square_name_title">"What surface does the square %s have here? Please, select a specific one."</string>
+    <string name="quest_surface_detailed_title">"What surface does this piece of road have? Please, select a specific one."</string>
+    <string name="quest_surface_detailed_square_title">"What surface does this square have here? Please, select a specific one."</string>
```

Comment:
Thinking about this, does it really have any advantages to mention "Please select a specific one"?

Otherwise, the strings from the normal surface quest could be reused to safe the translators some work.

Reply:
Less confusing in quest list in settings, less confusing why paved/unpaved is gone.

Reply:
Fair enough. Maybe rephrase to "What specific surface does the road %s have here?" though then? It's more compact.